mlir/test/Dialect/Vector/vector-contract-to-dot-transforms.mlir

```diff
@@ -169,13 +169,11 @@ func.func @extract_contract3(%arg0: vector<3xf32>,
 // CHECK:    %[[T21:.*]] = vector.insert %[[T20]], %[[T11]] [0, 1] : f32 into vector<2x2xf32>
 //
 // CHECK:    %[[T23:.*]] = vector.extract %[[A]][1] : vector<2xf32> from vector<2x2xf32>
-// CHECK:    %[[T24:.*]] = vector.extract %[[Bt]][0] : vector<2xf32> from vector<2x2xf32>
-// CHECK:    %[[T32:.*]] = arith.mulf %[[T23]], %[[T24]] : vector<2xf32>
+// CHECK:    %[[T32:.*]] = arith.mulf %[[T23]], %[[T2]] : vector<2xf32>
```

Comment:
With the new [Testing Guide for MLIR ](https://github.com/llvm/mlir-www/pull/216) in mind (soon to be merged), could you replace `T32` and `T41` with some meaningful names? Right now it's impossible to tell that these are columns of `B`.

Bonus points for updating all CHECK lines.

Reply:
The guide is helpful, thank you.